prototype/distributor/src/SessionData.java

```diff
@@ -1,15 +1,16 @@
 package prototype.distributor;
 
-import java.util.ArrayList;
-import prototype.distributor.UsersData;
 import prototype.protobuf.Session;
 
+import java.util.ArrayList;
+
 public class SessionData {
```

Comment:
This class needs threadsafety

Reply:
Added a synchronized tag to addClientUser. Resolved, maybe